src/rev_ai/topic_extraction_client.py

```diff
@@ -0,0 +1,50 @@
+from .generic_api_client import GenericApiClient
+from .models import TopicExtractionJob, TopicExtractionResult
+
+
+class TopicExtractionClient(GenericApiClient):
```

Comment:
I'm pretty happy with this class. I think it looks very clean, basically all the functionality is in the GenericApiClient and it works perfectly